client/cpp/kserve-api/README.md

```diff
@@ -0,0 +1,345 @@
+# KServe API usage samples
+
+OpenVINO Model Server 2022.2 release introduced support for [KServe API](https://github.com/kserve/kserve/tree/master/docs/predict-api/v2).
+
+## Before you run the samples
+
+### Clone OpenVINO&trade; Model Server GitHub repository and go to the top directory.
+```Bash
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server
+```
+
+### Start the Model Server Container with Dummy Model
+```Bash
+docker run --rm -d -v $(pwd)/src/test/dummy:/models -p 9000:9000 openvino/model_server:latest --model_name dummy --model_path /models --port 9000 
+```
+
+### Build client library and samples
+```Bash
+cd client/cpp/kserve-api
+cmake . && make
+```
+
+## GRPC Examples <a name="grpc-api"></a>
+
+
+## GRPC Examples with Dummy Model
+
+This section demonstrates inference on a simple model, which increments each provided value. 
+
+
+Once you finish above steps, you are ready to run the samples.
+
+### Run the Client to get server liveness <a name="grpc-server-live"></a>
+
+- Command
+
+```Bash
+./grpc_server_live --help
+Sends requests via KServe gRPC API to check if server is alive.
+Usage:
+  grpc_server_live [OPTION...]
+
+  -h, --help              Show this help message and exit
+      --grpc_address arg  Specify url to grpc service.  (default: 
```

Comment:
Maybe instead of arg use just argument name like in out other readme's, for example --grpc_addres GRPC_ADDRESS. This applies to all of the following examples